### PR TITLE
Separate WorkerManager::Gearman

### LINKED ISSUE #7: Remove or separate WorkerManager::Gearman
TheSchwartz is de-facto standard backend for WorkerManager, so Gearman is no longer used at @hatena. We should remove unnecessary dependencies like `Danga::Socket` for Gearman backend.

### PR DESCRIPTION
**Pull request by akiym**
Fix #7
Remove WorkerManager::Gearman and these dependencies. New distribution is here: https://github.com/hatena/WorkerManager-Gearman

### CODE REVIEW

File: META.json [hunk 1]
```diff
@@ -78,19 +78,21 @@
...
-      "hideoki <hideoki@hatena.ne.jp>",
-      "onishi <onishi@hatena.ne.jp>",
-      "Shinji Tanaka <stanaka@hatena.ne.jp>",
-      "wakabatan <wakabatan@localhost.localdomain>",
-      "mechairoi <ttsujikawa@gmail.com>",
       "Kentaro Kuribayashi <kentarok@gmail.com>",
-      "shiba_yu36 <shibayu36@gmail.com>",
-      "motemen <motemen@hatena.ne.jp>",
+      "Shinji Tanaka <stanaka@hatena.ne.jp>",
+      "Takumi Akiyama <t.akiym@gmail.com>",
```
Comment (astj) [inline META.json]: :hearts:

File: cpanfile [hunk 2]
```diff
@@ -1,17 +1,10 @@
...
-requires 'Class::Data::Inheritable';
 requires 'DBI';
-requires 'Danga::Socket';
 requires 'File::Pid';
+requires 'Module::Load';
 requires 'Parallel::ForkManager';
 requires 'Proc::Daemon';
 requires 'TheSchwartz';
 requires 'TheSchwartz::Simple';
-requires 'TheSchwartz::Worker';
```
Comment (astj) [inline cpanfile]: I believe TheSchwartz::Worker is still used at https://github.com/hatena/WorkerManager/blob/25bf2f435a9c325d98748ea64375db9b80379088/lib/WorkerManager/Worker/TheSchwartz.pm#L4 . Can we remove this line?

Comment (astj) [reply cpanfile]: I guess requiring `TheSchwartz` install `TheSchwartz:Worker`, but I prefer explicitly declaring this dependance, but I don't remember the best practice for cpanfile anymore :scream:

Comment (akiym) [reply cpanfile]: Yes `TheSchwartz` (distribution) has `TheSchwartz::Worker` (module), we don't need to declare in cpanfile.

**Comment by akiym** [comment]
🚶🚶🚶

**Pull request merged**